Assets/MixedRealityToolkit/InputSystem/Focus/FocusProvider.cs

```diff
@@ -21,7 +21,7 @@ namespace Microsoft.MixedReality.Toolkit.InputSystem.Focus
     public class FocusProvider : MonoBehaviour, IMixedRealityFocusProvider
     {
         private IMixedRealityInputSystem inputSystem = null;
-        public IMixedRealityInputSystem InputSystem => inputSystem ?? (inputSystem = MixedRealityManager.Instance.GetManager<IMixedRealityInputSystem>());
+        public IMixedRealityInputSystem InputSystem => inputSystem ?? (MixedRealityManager.Instance.ActiveProfile.EnableInputSystem ? inputSystem = MixedRealityManager.Instance.GetManager<IMixedRealityInputSystem>() : null);
```

Comment:
The Focus provider relies on our implementation of the input system. So if it isn't there, this should really break.

Reply:
Bad @StephenHodgson 
If we put something in a scene, then we should either remove it or ensure it doesn't break if something we provide is turned off.  So no, it shouldn't break.  At worst you should get a proper warning

Reply:
I thought I had cleaned this up better.

Do you have a proper repro?